tests/test_auth.py

```diff
@@ -4,19 +4,20 @@
 
 from .conftest import __tests__, raises
 
+default_auth_base_error_message = "default auth: cannot configure default credentials, please check https://github.com/databricks/databricks-sdk-py/tree/main#default-authentication-flow to configure credentials for your preferred authentication method"
```

Comment:
Thanks for the PR. I think @hectorcast-db was working on better handling, not sure if there are intersections but mentioning for viz.

Also, if going ahead with this, we would have to do this change in all SDKs for consistency.

Reply:
I was trying to fix the error messages coming from the API to us (which we display to the users). There are probably cases were both messages are displayed, but in general is a different error message.